packages/@aws-cdk/aws-stepfunctions-tasks/test/emr/emr-set-cluster-termination-protection.test.ts

```diff
@@ -107,8 +107,8 @@ test('Set termination protection with static ClusterId and TerminationProtected
     },
     End: true,
     Parameters: {
-      'ClusterId': 'ClusterId',
-      'TerminationProtected.$': '$.TerminationProtected',
+      ClusterId: 'ClusterId',
+      TerminationProtected: true,
```

Comment:
we don't typically modify existing tests as they can mask regressions.
Is there a reason why this needed to change?

related: the test name no longer maps to the behaviour it's testing

Reply:
`TerminationProtected` should be boolean rather than string, so I believe this test was invalid prior to my changes? I agree it doesn't match the name anymore, let me remedy that.